### DISCUSSION TITLE
Automation using wind speed will not trigger a switch.

### DISCUSSION THREAD

**Discussion by christunited** [home-assistant/frontend#21982] [Automations / scripts / scenes] [answered]
I also ran into this problem trying to create automation using a thermostat to turn on a switch when temperature reaches a set point.
I will not trigger switch when temp reached. Same with the wind speed, it will not activate when wind speed is reached.
![Screenshot 2024-09-14 at 06-16-23 History – Home Assistant](https://github.com/user-attachments/assets/82c352b1-b8ee-45a0-b554-ae24473526e5)
![Screenshot 2024-09-14 at 06-15-37 Overview – Home Assistant](https://github.com/user-attachments/assets/6a536755-83da-4670-ae8f-2e5281cb0df5)
![Screenshot 2024-09-14 at 06-15-14 Settings – Home Assistant](https://github.com/user-attachments/assets/d0b40777-5163-4563-a95a-cfd6314c62e9)
![Screenshot 2024-09-14 at 06-13-21 Settings – Home Assistant](https://github.com/user-attachments/assets/c3ba7eb5-bd81-4c69-baab-fab9964baed8)

**Comment by karwosts** [comment, marked as answer]
You're probably better off asking a question on the forums (including your full automation yaml). This isn't really a user support area. 

https://community.home-assistant.io/
